vignettes/download_github_comments.Rmd

```diff
@@ -12,10 +12,19 @@ vignette: >
 
 # Introduction
 
-
 In this Vignette, we show how to download GitHub comments from both Issue and Pull Requests. This data may be useful if development communication occurs through issues or pull requests in GitHub in addition or instead of mailing lists. For details on how to merge various communication sources, see the `reply_communication_showcase.Rmd` notebook. The parsed communication table from GitHub comments by `parse_github_comments` has the same format as the mailing list and JIRA comments tables. In turn, the function expects as input the table generated at the end of this notebook.
```

Comment:
> "This data may be useful if development communication occurs through issues or pull requests in GitHub in addition or instead of mailing lists"

is not a well-worded or clear sentence. If I am understanding it correctly, maybe something along the lines of:
> Downloading comments may be useful if communication during development occurs through issue or pull requests in addition, or in lieu of, mailing lists.

Clarifying ideas you're attempting to convey, especially with all of these nouns in one sentence, by elaborating terms like "development communication" to something understandable, rewording "addition or instead of" helps, or even what "data" is (in this case comments). This applies to all explanations, pretend as if the user has little-to-no knowledge of the functionality and terminology. In this case, throughout the notebook, possibly think of changing "development communication" to the self-explanatory "communication during development", or just explicitly explain it. The reason I say this is because my tired brain read "development communication" as "communication that's developing", and we need to find less ways to confuse our users.

Also, please show us or put a sample block of the expected parsed communication table or the expected input table- we want to reduce the amount of notebook jumping as much as possible